src/actions/profile-view.js

```diff
@@ -44,21 +55,131 @@ import type {
 import type { Transform } from '../types/transforms';
 
 /**
- * The actions that pertain to changing the view on the profile, including searching
- * and filtering. Currently the call tree's actions are in this file, but should be
- * split apart. These actions should most likely affect every panel.
+ * This file contains actions that pertain to changing the view on the profile, including
+ * searching and filtering. Currently the call tree's actions are in this file, but
+ * should be split apart. These actions should most likely affect every panel.
+ */
+
+/**
+ * Select a call node for a given thread. An optional call node path can be provided
+ * to expand child nodes beyond the selected call node path.
```

Comment:
Maybe mention here that `optionalExpandedToCallNodePath`, if specified, must be a descendant call node of `selectedCallNodePath`.